recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,16 @@
+YEAR:
+  - 2017
+VER:
+  - 15
+# the VS update version.  This is the middle digit in the version reported in the VS help->about UI.  It is perhaps a more readily referenceable number.
+update_version:
+  - 9
+# this is the version number reported by cl.exe
+cl_version:
+  - 19.16.27032.1
+# this version follows the version number reported for the vcruntim140.dll file.
+#     This will be in a folder like C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\VC\Redist\MSVC\14.15.26706\x64\Microsoft.VC141.CRT
+#     Note the 14.15.26706 in there - that's what we want, but in case their folder scheme changes, it's better to base it on the version of the file itself.
+runtime_version:
+  - 14.16.27012
```

Comment:
Shouldn't this be `14.16.27032` if `cl_version` is `19.16.27032`?

Reply:
I have no idea. :) So if you think so, yes!